job-dsls/jobs/osbs/bamoe_osbs_seed_cloud_jobs.groovy

```diff
@@ -9,7 +9,7 @@ String jobDescription = 'Job responsible for seed jobs to building bamoe openshi
 def prodComponent = [
         'bamoe-businesscentral', 'bamoe-businesscentral-monitoring',
         'bamoe-controller', 'bamoe-kieserver', 'bamoe-smartrouter',
-        'bamoe-process-migration']
+        'bamoe-process-migration', 'rhpam-dashbuilder']
```

Comment:
I suppose it should be bamoe-dashbuilder here?

Reply:
oops, yes :)

Reply:
updated